### PR TITLE
Merge updates from ocm-plus repo

### PR DESCRIPTION
**Pull request by JustinKuli**
Brings in commits up to the current "release-2.4" branch from the other repo.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -54,6 +54,8 @@ ifneq ($(KIND_VERSION), latest)
 else
```
Comment (dhaiducek) [inline Makefile:54]: There's an extra `KBVERSION` assignment on line 51.

File: Makefile [hunk 2]
```diff
@@ -199,7 +201,7 @@ kind-delete-cluster:
 
 install-crds:
 	@echo installing crds
-	kubectl apply -f https://raw.githubusercontent.com/open-cluster-management/governance-policy-propagator/main/deploy/crds/policy.open-cluster-management.io_policies_crd.yaml
+	kubectl apply -f https://raw.githubusercontent.com/open-cluster-management/governance-policy-propagator/main/deploy/crds/policy.open-cluster-management.io_policies.yaml
```
Comment (dhaiducek) [inline Makefile:204]: Hmmm...this should probably point to the `-io` org? But now I'm wondering if there are links in other Makefiles that need to be updated...

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Thanks! Based on a [previous thread](https://coreos.slack.com/archives/GUQS3CS5A/p1633530713386900?thread_ts=1633530713.386900), Let's get rid of the `vendor` folder. (I don't see it upstream or midstream, so pulling in some more commits might do it?)

File: Makefile (hunk 2 above)
Comment (JustinKuli) [reply Makefile:204]: Exactly right, I was planning on going through in another set of PRs to address things like this.

File: Makefile (hunk 1 above)
Comment (JustinKuli) [reply Makefile:54]: Oh, nice catch!

File: .github/workflows/kind.yml [hunk 3]
```diff
@@ -4,11 +4,11 @@ on:
   push:
     branches:
       - main
-      - release-2.3
+      - release-2.[3-9]
```
Comment (dhaiducek) [inline .github/workflows/kind.yml:7]: I think we might want to remove this artifact along with the `-io` links in the Makefiles.

**Review by dhaiducek — APPROVED** [review]

File: .github/workflows/kind.yml (hunk 3 above)
Comment (JustinKuli) [reply .github/workflows/kind.yml:7]: Agree

**Pull request merged**